migrations/2021-03-09-171136_split_user_table_2/up.sql

```diff
@@ -0,0 +1,391 @@
+-- Person
```

Comment:
Feel my pain by looking at both this and `down.sql`. Writing `down.sql` was probably worse than making this one too... every change you need to put in opposite order and do backwards.

Reply:
Almost 800 lines of migrations, and I dont even understand half of it. Is there no easier way to do this?

Reply:
Nope.